model/src/w3timemd.F90

```diff
@@ -1313,6 +1316,21 @@ SUBROUTINE D2J(DAT,JULIAN,IERR)
 
     JULIAN = -HUGE(99999)                  ! this is the date if an error occurs and IERR is < 0
 
+    ! Special case for 360 day climate calendar; return a pseudo-Julian day
+    ! Assumes a reference date of 1800-01-01 00:00:00
+    IF( CALTYPE .EQ. "360_day" ) THEN
+      JULIAN = (YEAR - 1800) * 360.0 +     &  ! Years since 1800
+            (MONTH - 1) * 30.0  +          &  
+            (DAY - 1) +                    & 
+            HOUR / 24.0_8 +                &
+            MINUTE / 1440.0_8 +            &
+            SECOND / 86400.0_8
+
+      IERR = 0
+      RETURN
+    ENDIF
+
+    ! Standard/Gregorian calenday - return standrd Julian day calculation:
```

Comment:
standard

Reply:
You've got a good eye for spotting my typos! Thanks. :)